controllers/capa_controller.go

```diff
@@ -85,13 +86,26 @@ func (r *CAPAClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	if accountID == "" {
 		logger.Error(err, "Unable to extract Account ID from ARN")
-		return ctrl.Result{}, microerror.Mask(fmt.Errorf("Unable to extract Account ID from ARN %s", string(arn)))
+		return ctrl.Result{}, microerror.Mask(fmt.Errorf("unable to extract Account ID from ARN %s", string(arn)))
+	}
+
+	// Fetch config map created by cluster-apps-operator
+	clusterValues := &v1.ConfigMap{}
```

Comment:
I moved to the controllers package the fetching `cluster-values` configmap, adding/deleting a finalizer to it